.github/workflows/release.yml

```diff
@@ -0,0 +1,118 @@
+# -------------------------------------------------------------------------------------
+#
+# Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# --------------------------------------------------------------------------------------
+
+name: 🚀 Release
+
+on:
+    push:
+        branches: ["main"]
+        paths:
+            - "applications/*/**"
+            - "connections/*/**"
+    workflow_dispatch:
+        inputs:
+            integration_type:
+                description: Select the integration type for which you want the artifacts to be released
+                type: choice
+                options:
+                    - applications
+                    - connections
+                required: true
+            integration_name:
+                description: Enter the names of the integrations to be released (e.g., salesforce,zoom)
+                required: true
+            release_type:
+                description: Select the release type
+                type: choice
+                options:
+                    - major
+                    - minor
+                    - patch
+                required: true
+
+env:
+    INTEGRATION_TYPES: applications,connections
+    MAIN_VERSION: v1.0.0
```

Comment:
Should this be static?

Reply:
Renamed the variable for more clarity. This is the initial version of the template.